Makefile

```diff
@@ -109,27 +109,28 @@ ifeq ($(GCP_ONLY),true)
 		--zone $(GKE_ZONE) \
 		--project $(GCP_PROJECT)
 endif
-	GCP_ONLY=$(GCP_ONLY) go test -v $(REPOPATH)/integration -timeout 15m $(INTEGRATION_TEST_ARGS)
+	kubectl get nodes -oyaml
+	GCP_ONLY=$(GCP_ONLY) go test -v $(REPOPATH)/integration -timeout 20m $(INTEGRATION_TEST_ARGS)
```

Comment:
I don't fully understand how this relates to your comment:

> Increase the duration of the test suite to increase the chance of running all the tests when one goes wrong.

Do you mean that this gives enough time for the remaining tests to finish, so that they are not marked as failed?

Reply:
Exactly! It's so better said!